lib/appmap/event.rb

```diff
@@ -36,20 +36,38 @@ def display_string(value)
             '*Error inspecting variable*'
           end
 
-          value_string = \
+          value_string = custom_display_string(value) || default_display_string(value)
+
+          (value_string||'')[0...LIMIT].encode('utf-8', invalid: :replace, undef: :replace, replace: '_')
+        end
+
+        protected
+
+        def custom_display_string(value)
+          case value
+          when File
+            "#{value.class}[path=#{value.path}]"
+          when Net::HTTP
+            "#{value.class}[#{value.address}:#{value.port}]"
+          when Net::HTTPGenericRequest
+            "#{value.class}[#{value.method} #{value.path}]"
+          end
+        rescue StandardError
```

Comment:
`StandardError` is what gets caught by a bare `rescue`, isn't it? Is there some reason it needs to be mentioned explicitly here?

Reply:
Rubucop :-(